tests/utils/helper.py

```diff
@@ -23,13 +25,15 @@ def run_make_shell(args=None, tmpdir=None, **kwargs):
     return make_process
 
 
-def run_ansible_playbook(playbook, env=None, tmpdir=None):
+def run_ansible_playbook(playbook, env=None, basedir=None, tmpdir=None):
     env_vars = dict(os.environ)
     env_vars.setdefault('ANSIBLE_FORCE_COLOR', "true")
     if env:
         env_vars.update(env)
     command = "ansible-playbook playbooks/{}".format(playbook)
-    return run_make_shell(args=command, env=env_vars, tmpdir=tmpdir)
+    kwargs = dict(basedir=basedir, tmpdir=tmpdir)
```

Comment:
I'm not too fond of creating a `dict` by passing two `kwargs` to the `dict` constructor, then updating two (different) keys using `update` in the second line (also using the not-so-common `kwargs`-based approach), then passing that dict as `**kwargs` to some function.

I really think something like

```python
return run_make_shell(
    args=command,
    env=env_vars,
    basedir=basedir,
    tmpdir=tmpdir,
)
```

is much more readable. Though granted, the current code is not 'wrong' per se.